models/POP/readme.rst

```diff
@@ -250,19 +212,6 @@ integration.
 Other files needed for assimilation
 -----------------------------------
 
-Creating an initial ensemble
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Karspeck et al. (2013) [3]_ find that an ensemble of 1 January model states
-selected from a multi-decade free-running integration of POP2 can be used as an
-initial ensemble.
```

Comment:
My motivation here was that our users don't have to create the initial ensemble anymore or use this script since they can just get it from the GDEX now. So I figured we didn't need this whole bit of text. But I agree, we should leave the reference in. Fixing now.

Reply:
Yes, I agree this reference should be left in because that's actually how the initial ensemble was generated, so the reference should be kept in.